---
geometry_msgs/Point result
